doc/topics/releases/templates/3006.0.md.template

```diff
@@ -14,6 +14,25 @@ be provided for supported operating systems. See [Upgrade to onedir](https://doc
 in the [Salt Install Guide](https://docs.saltproject.io/salt/install-guide/en/latest) for information about upgrading from the classic packages to the onedir
 packages.
 
+## Linux Packaging Salt Master Salt User and Group
+
+The linux Salt Master packages will now add a Salt user and group
+by default. The packages will add the ``user: salt`` config option
+to the Salt Master config. The Salt Master service will run as this
+user. If you do not want to run the Salt Master as a different user
+you can remove the ``user: salt`` config from /etc/salt/master or change
+the user to the desired user to run the Salt Master.
+
+Since the Salt Master is running as a different user, you will need
+to ensure you set the owner and group to ``salt`` for your file_roots
+and pillar_roots.
```

Comment:
```suggestion
Since the Salt Master is running as a different user, you will need
to ensure you set the owner and group to ``salt`` for your file_roots
and pillar_roots(commonly `/srv/salt` and `/srv/pillar`).
```
?